alica_tests/src/test/test_success.cpp

```diff
@@ -10,48 +10,14 @@
 #include <communication/AlicaDummyCommunication.h>
 #include <engine/AlicaTimer.h>
 #include <engine/logging/AlicaDefaultLogger.h>
+#include <test_alica.h>
 
 #include <gtest/gtest.h>
 
 namespace alica::test
 {
 
-class TestSuccessFixture : public ::testing::Test
-{
-public:
-    virtual void SetUp() override
-    {
-        // Path to test configs set by CMake
-        std::string path;
-#if defined(PLANS)
-        path = PLANS;
-        path += "/src/test";
-#endif
-        _tc = std::make_unique<TestContext>("hairy", std::vector<std::string>{path + "/etc/"}, "Roleset", "TestMasterPlan", true, 1);
-        ASSERT_TRUE(_tc->isValid());
-        const YAML::Node& config = _tc->getConfig();
-
-        _tc->setCommunicator<alicaDummyProxy::AlicaDummyCommunication>();
-        _tc->setTimerFactory<alica::AlicaSystemTimerFactory>();
-        _tc->setLogger<alica::AlicaDefaultLogger>();
-
-        LockedBlackboardRW(_tc->editGlobalBlackboard()).set("worldmodel", std::make_shared<alicaTests::TestWorldModelNew>(_tc.get()));
```

Comment:
this is not used anywhere? I remember the world model was used to get access to the test context at some point